site/src/sphinx/server-basics.rst

```diff
@@ -143,6 +143,29 @@ You might be interested in decorating a service using other decorators, for exam
 
 You can also use an arbitrary object that's annotated by the ``@Path`` annotation using ``annotatedService()``.
 
+Path pattern
+------------
+
+You can use the following path patterns to map the path of an HTTP request on a service or decorators.
+
++------------------------------------------+---------------------------------------+
+| Pattern                                  | Example                               |
++==========================================+=======================================+
+| ``Plain``                                | /login                                |
++------------------------------------------+---------------------------------------+
+| ``Curly-brace style path variable``      | /users/{userId}                       |
```

Comment:
- I guess we don't need to use monospace fonts on the first column? Monospace font has to be used for the second column in my opinion.
- variables